CONTRIBUTING.md

```diff
@@ -97,6 +95,41 @@ committing your changes. Most editors have plug-ins that do this automatically.
 Pull request descriptions should be as clear as possible and include a reference
 to all the issues that they address.
 
+### Testing
+
+Submit tests for your changes. Go has a great test framework built in; use it!
```

Comment:
I changed "submit unit tests" to "submit tests" because there currently aren't any unit tests, only integration/regression tests.